backends/official_pyladies_about_scraping.py

```diff
@@ -0,0 +1,36 @@
+# This file scrapes the official pyladies 
+# website 'https://pyladies.com/about/'
+# and assigns about info text to 
+# str type variable 'about_info'
+# which can be further used in our website
+
+# in case the structure is of official pyladies
+# website is changed in future, link to the website
+# as str is assigned to 'about_info' variable
+
+import requests
+from bs4 import BeautifulSoup   # for web scraping
+
+official_pyladies_url="https://pyladies.com/about/"
+
+def get_official_pyladies_about_info(url=official_pyladies_url):
```

Comment:
Let's remove this `official` word, it is not really relevant and it takes space.